frontend/src/app/moduls/register/user-registration-form/user-registration-form/user-registration-form.component.ts

```diff
@@ -0,0 +1,87 @@
+import {
+  Component, EventEmitter, OnInit, Output
+} from '@angular/core';
+import {
+  AbstractControl, FormControl, FormGroup, ValidationErrors, ValidatorFn, Validators
+} from '@angular/forms';
+import { UserRegisterDto } from 'src/app/models/auth/user-register-dto';
+
+@Component({
+  selector: 'app-user-registration-form',
+  templateUrl: './user-registration-form.component.html',
+  styleUrls: ['./user-registration-form.component.sass']
+})
+export class UserRegistrationFormComponent implements OnInit {
+  public registrationForm!: FormGroup;
+  @Output() isSuccess = new EventEmitter<boolean>();
+  @Output() isRequiredError = new EventEmitter<boolean>();
+  @Output() newUser = new EventEmitter<UserRegisterDto>();
+
+  ngOnInit(): void {
```

Comment:
There's no need to explicitly write return type. TS compiler understands the type in 99% of cases